evap/staff/tests/test_tools.py

```diff
@@ -219,6 +222,52 @@ def test_do_nothing_if_test_run(self):
         self.assertEqual(len(messages), 4)
 
 
+class RemoveUserDueToInactivity(TestCase):
+    def test_remove_user_due_to_inactivity(self):
+        inactive_user = baker.make(UserProfile, last_name="Inactive User")
+        inactive_evaluation = baker.make(
+            Evaluation,
+            state=Evaluation.State.PUBLISHED,
+            vote_start_datetime=datetime.today()
+            - timedelta(days=settings.PARTICIPATION_DELETION_AFTER_INACTIVE_MONTHS + 1),
+            vote_end_date=datetime.today() - timedelta(days=settings.PARTICIPATION_DELETION_AFTER_INACTIVE_MONTHS + 1),
+            participants=[inactive_user],
+        )
+
+        semester = inactive_evaluation.course.semester
+        semester.archive()
+
+        messages = remove_inactive_participations(inactive_user)
+        self.assertTrue(inactive_user.can_be_marked_inactive_by_manager)
+        self.assertEqual(len(messages), 1)
+
+    def test_do_not_remove_user_due_to_inactivity(self):
+        active_user = baker.make(UserProfile, last_name="Active User")
+
+        baker.make(Evaluation, state=Evaluation.State.PUBLISHED, participants=[active_user])
```

Comment:
This is interesting. I don't think there is an actual guarantee here about what will be the `vote_start_datetime` and `vote_end_date` of this evaluation. Thus, this test might (randomly) fail, if model_bakery decides to give us dates way out in the past.

Since the test actually relies on the values, I think we will have to set them.